runelite-client/src/main/java/net/runelite/client/plugins/itemidentification/ItemIdentification.java

```diff
@@ -360,6 +360,58 @@ enum ItemIdentification
 	DWARF_WEED_POTION(Type.POTION, "Dwarf", "D", ItemID.DWARF_WEED_POTION_UNF),
 	TORSTOL_POTION(Type.POTION, "Torstol", "TOR", ItemID.TORSTOL_POTION_UNF),
 
+	// Barbarian Potions
```

Comment:
Can we interleave barbarian potions in with regular potions so that it's obvious the abbreviations match?

Reply:
Interleaving is inserting something in a regular, alternating form. In this case, the first example (attack, attack mix, strength, strength mix, etc.) is what I had in mind, which matches your most recent push, so that looks good.